src/org-manager/containers/pending-pbas/pending-pbas.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Observable, Subscription } from 'rxjs';
+import { filter, takeWhile } from 'rxjs/operators';
+
+import { OrganisationVM } from '../../models/organisation';
+import { PbaService } from '../../services';
+import * as fromStore from '../../store';
+
+@Component({
+  selector: 'app-pending-pbas',
+  templateUrl: './pending-pbas.component.html',
+})
+export class PendingPBAsComponent implements OnInit, OnDestroy {
+  public static PENDING_STATUS: string = 'pending';
+
+  public orgs$: Observable<OrganisationVM[]>;
+  public orgsWithPendingPBAs: any[];
+  public organisationFilter: string;
+  public get pendingPBAsCount(): number {
+    if (this.orgsWithPendingPBAs) {
+      return this.orgsWithPendingPBAs.reduce((count, org) => {
+        return count + org.pbaNumbers.length;
+      }, 0);
+    }
+    return -1;
+  }
+
+  private getActiveLoadedSubscription: Subscription;
+
+  constructor(
+    private readonly store: Store<fromStore.OrganisationRootState>,
+    private readonly pbaService: PbaService,
+    private readonly router: Router // Temporarily here for testing purposes.
+  ) {}
+
+  public ngOnInit(): void {
+    const status = this.getStatus();
+    this.pbaService.getPBAsByStatus(status).subscribe((pendingPBAs: any[]) => {
```

Comment:
can this be taken out of the ngOnInit() lot going on in this method. Also the subscribe is the old (soon to be deprecated) style, should now be

`.subscribe({ next: () => {}, error: () => {} }`